flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/ThreadInfoSampleService.java

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
For such helper commits, it's always good to state when they are used. The `ThreadInfoSampleService` is later injected into ... to ....